src/components/ClientForm.vue

```diff
@@ -290,12 +331,39 @@ export default {
       vm.companyName = vm.company.get("name");
       vm.completeCompany();
     },
+    newUniqueAttribute() {
+      const vm = this;
+      vm.bringUpNewKeyPrompt = false;
+      var newAtt = new AV.Object("ClientAttributes");
+      newAtt
+        .set("client", vm.client)
+        .set("key", vm.newKey)
+        .set("value", vm.newValue)
+        .save()
+        .then(
+          function() {},
+          function(error) {
+            alert(error);
+          }
+        );
+      vm.newKey = "";
+      vm.newValue = "";
+      vm.uniqueAttributes = vm.getUniqueAtts();
```

Comment:
These should go inside the callback function in the `.then` above.

Always keep in mind that JavaScript [handles requests asynchronously](https://www.pluralsight.com/guides/introduction-to-asynchronous-javascript). It's not like Python which waits until a response for the request made by a line is obtained and then move on to the next line.

Reply:
Fixed